### DISCUSSION TITLE
Black screen issue

### DISCUSSION THREAD

**Discussion by notvu** [jordanbtucker/totk-no-loot-boxes#12] [Q&A] [answered]
Whenever I start the game on yuzu with this mod enabled the game goes completely black. It still says that the game is running in 30fps and so on, but I just can't see anything. Whenever I disable the mod, the game runs fine. I'm on update 1.1.0 of TotK, on yuzu 1439. I do use other mods but I have tried to disable all of them and only enabling NoLootBoxes and the issue persists. I would be very thankful for any help!

**Comment by jordanbtucker** [comment]
How long did you wait at the black screen? I noticed it did that for me the first time, and I just let it run for a bit before the game started. Then it was fine every time after that.

If you could post the yuzu log files, that would help too.

**Reply by jochip** [reply]
I'm having the same issue, i emptied my mods folder except for the NoLootboxes mod and left it alone for 10mins to no avail, Yuzu just stayed on the black screen

I've provided my log files [included both bc not sure which one is needed]
[yuzu_log.txt](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11521503/yuzu_log.txt)
[yuzu_log.txt.old.txt](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11521504/yuzu_log.txt.old.txt)

**Reply by jordanbtucker** [reply]
Thanks. Your logs say you're running TotK v1.0.0. This mod requires v1.1.0 or v1.1.1.

**Reply by jochip** [reply]
oh yeah sorry i should've clarified, it is 1.1.0, i use a pre-updated NSP (i downloaded it as 1.1.0 instead of getting a 1.0.0 and then updating it) but i'll retry it with a manually updated one

![image](https://github.com/jordanbtucker/totk-no-loot-boxes/assets/134120024/259a4a2f-f75b-49cb-b78a-568ee8ffa81b)

**Reply by jordanbtucker** [reply]
It's odd that the filename says v1.1.0 but the version field says v1.0.0. I guess I wouldn't trust that file.

**Reply by jochip** [reply]
update: i just reloaded the shader cache and it seemed to work (i downloaded someone else's cache so it might have been corrupted), but i'll do the manual update anyways just in case it gives me any issues i just haven't come across yet, thank you!

**Comment by jordanbtucker** [comment]
@notvu Another user posted their logs and it said they were running TotK v1.0.0. This mod requires v1.1.0 or v1.1.1. Can you please check your version of TotK?

**Reply by notvu** [reply]
Sorry for the late reply, I haven't been online. I double checked and my version is 1.1.0. I would like to add my log files but I have no idea how. :)

EDIT: I think I managed to find them!
[yuzu_log.txt](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11522554/yuzu_log.txt)

**Reply by jordanbtucker** [reply, marked as answer]
Thanks for the logs. Please try this:

NOTE: These steps will delete your shader cache, which means yuzu will need to recompile them while running the game again. This will result in slower performance and stutters the first time you visit areas and encounter new textures and effects, but it should run normally again the next time you encounter them.

1. Open yuzu.
2. Right click Tears of the Kingdom > Remove > Remove All Pipeline Caches.
3. Click Yes.
4. Click OK.
5. Start Tears of the Kingdom.

If that doesn't fix it, then try this:

1. Open yuzu.
2. Right click Tears of the Kingdom > Remove > Remove All Pipeline Caches. (Yes, do this step again even though you just did it.)
3. Click Yes.
4. Click OK.
5. Right click Tears of the Kingdom > Remove > Remove All Installed Contents.
6. Click Yes.
7. Click OK on any windows and ignore any errors.
8. Right-click Tears of the Kingdom > Properties.

If the Version field says 1.0.0, follow these steps. Otherwise, skip them.

1. Click Cancel.
2. Click File > Install Files to NAND...
3. Select the Tears of the Kingdom v1.1.0 Update NSP file.
4. Click Open.
5. Click Install.

Finally, start Tears of the Kingdom.

**Reply by notvu** [reply]
It worked thanks!

**Reply by jordanbtucker** [reply]
Did you only have to remove the pipeline caches, or did you have to reinstall the updates too?

**Reply by notvu** [reply]
I had to reinstall the updates and all that, though on second launch of the game (since reinstall) multiple of my mods have slightly broken. The black screen caused by this mod is back, and other mods have stopped working in certain ways. Such as one specific UI that my dualshock UI mod should replace hasn't been changed, or that the faster fast travel mod isn't working at all.

 Edit: I reinstalled the mods that were wonky and they work! Including the NoLootBoxes mod! I guess TotK modding is a bit jank right now but where is the fun without a challange? :)
 
 Thanks for the help!
